Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -1043,6 +1043,18 @@ bool HTMLInputElement::isTextType() const
     return m_inputType->isTextType();
 }
 
+bool HTMLInputElement::supportsWritingSuggestions() const
```

Comment:
We really need to refactor how this works. I get the feeling we're being quite inconsistent about this across the code base. (Not a blocker for this patch though.)